consumer-backend/package.json

```diff
@@ -29,6 +29,7 @@
     "express": "^4.18.1",
     "kafkajs": "^2.2.2",
     "kube-service-bindings": "^2.1.0",
+    "simple-oauth2": "^4.3.0",
```

Comment:
Is it possible to use https://www.npmjs.com/package/openid-client, that is what we will be recommending for use with Red Hat SSO